### ISSUE TITLE
Przekopiowac parametry (pomijajac 4 kanal) z papera i sprawdzic wyniki

### ISSUE THREAD

**Issue by rojberr** [GHOST-Science-Club/tree-classification-irim#158]
> 4.5 Classification from aerial images
VHR aerial imagery is more commonly available than ALS data, and it is important to evaluate how the two modalities compare. Therefore, we also train and evaluate a basic image classifier on aerial imagery. We adopt the experimental context reported by creators of TreeSatAI for their own baseline [[5](https://arxiv.org/html/2404.12064v2#bib.bib5)]: a ResNet18 [[34](https://arxiv.org/html/2404.12064v2#bib.bib34)] encoder with a batch size of 32, optimized with Adam with a cyclic learning rate (0.00005-0.001 range, half-period of 13630 steps). The model is pretrained on ImageNet and we adapt data processing accordingly: images are resized to 224 × 224 pixels, the model is adapted to receive the additional infrared channel by duplicating the weights for the red channel, and color channels are normalized using ImageNet statistics. We use common data augmentations: random rotations, horizontal and vertical flips, color jitter (p=0.5), random cropping (p=0.5), channel dropout (p=0.05).
Training is parallelized as before with 3 GPUS, and we scale the learning rate range linearly to account for the effective batch size. Since the model is pretrained, it converges in a few training epochs and then quickly plateaus. This narrows the window of good models, and therefore we validate the model more frequently, i.e., every tenth of the training set. Training takes 4 minutes per epoch on average.

Punkt 4.5 z publikacji: https://arxiv.org/html/2404.12064v2

**Comment by rojberr** [comment]
https://github.com/GHOST-Science-Club/tree-classification-irim/blob/2d7361f8b70560ec912115f8e4b52523af9f3202/src/transforms.py#L19

https://lightning.ai/docs/pytorch/stable/notebooks/lightning_examples/augmentation_kornia.html

**Comment by rojberr** [comment]
We could cont augemented data (scheduler, steps) and create a similiar count.
